rubyparser.h

```diff
@@ -615,6 +619,46 @@ typedef struct RNode_LIT {
     VALUE nd_lit;
 } rb_node_lit_t;
 
+typedef struct RNode_INTEGER {
+    NODE node;
+
+    char* val;
+    int tminus;
```

Comment:
What is `t` of `tminus`?
`t` of `tMINUS` is stand for "token", but this is not a token.